backend/company_api.py

```
@@ -936,7 +936,154 @@ async def get_public_doctor_report():
+                            tech_stack.extend(w.inferred_stack.frameworks or [])
+                            tech_stack.extend(w.inferred_stack.languages or [])
+                            tech_stack.extend(w.inferred_stack.cms or [])
+                    # Get system types
+                    detected = await store.list_detected_systems(company_id)
+                    system_types = list({d.system_type for d in detected})
+
+                    # Get specialist families
+                    specialists = await store.list_specialists(company_id)
+                    specialist_families = list({s.family for s in specialists})
+            except Exception:
+                pass
```

Ruff (0.15.15)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)